bzr/meta.yaml

```diff
@@ -0,0 +1,18 @@
+package:
+  name: bzr
+  version: 2.6.0
+
+source:
+  fn: bzr-2.6.0.tar.gz
+  url: https://launchpad.net/bzr/2.6/2.6.0/+download/bzr-2.6.0.tar.gz
+  md5: 28c86653d0df10d202c6b842deb0ea35
+
+build:
+  number: 1
```

Comment:
I think I have a version on my public channel that has build number 1 (I
think I used zero testing), is it worth worrying about the global build
number?

On Thu, May 7, 2015, 19:28 Carlos Cordoba <notifications@github.com> wrote:

> In bzr/meta.yaml
> <https://github.com/conda/conda-recipes/pull/259#discussion_r29906017>:
>
> > @@ -0,0 +1,18 @@
> > +package:
> > +  name: bzr
> > +  version: 2.6.0
> > +
> > +source:
> > +  fn: bzr-2.6.0.tar.gz
> > +  url: https://launchpad.net/bzr/2.6/2.6.0/+download/bzr-2.6.0.tar.gz
> > +  md5: 28c86653d0df10d202c6b842deb0ea35
> > +
> > +build:
> > +  number: 1
>
> Since this is a new package, build number must be 0. In other words, you
> don't need to add a build/number section for new packages
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/conda/conda-recipes/pull/259/files#r29906017>.
>